source/update.rst

```diff
@@ -135,3 +159,79 @@ Manual update
+the newer versions of Percona XtraDB Cluster within the method named *Smart
+Updates*.
+
+To have this upgrade method enabled, make sure that the ``updateStrategy`` key
+in the ``deploy/cr.yaml`` configuration file is set to ``SmartUpdate``.
+
+When automatic updates are enabled, the Operator will carry on upgrades
+according to the following algorithm. It will query a special *Version Service* 
+server at scheduled times to obtain fresh information about version numbers and
+valid image paths needed for the upgrade. If the current version should be
+upgraded, the Operator updates the CR to reflect the new image paths and carries
+on sequential Pods deletion in a safe order, allowing StatefulSet to redeploy
+the cluster Pods with the new image.
+
+The upgrade details are set in the ``upgradeOptions`` section of the 
+``deploy/cr.yaml`` configuration file. Make the following edits to configure
+updates:
+
+#. Set the ``apply`` option to one of the following values:
+
+   * ``Recommended`` - automatic upgrades will choose the most recent version
+     of software flagged as Recommended,
+   * ``Latest`` - automatic upgrades will choose the most recent version of
+     the software available,
+   * *specific version number* - will apply an upgrade if the running PXC
+     version doesn't match the explicit version number with no future upgrades
+     (version numbers are specified as ``5.7.26-31.37``, ``5.7.27-31.39``,
+     ``5.7.28-31.41``, ``5.7.29-31.43``, etc.),
+   * ``Never`` or ``Disabled`` - disable automatic upgrades
+     
+     .. note:: When automatic upgrades are disabled by the ``apply`` option, 
+        Smart Update functionality will continue working for changes triggered
+        by other events, such as updating a ConfigMap, rotating a password, or
+        changing resource values.
+
+#. Make sure the ``versionServiceEndpoint`` key is set to a valid Version
+   Server URL (otherwise Smart Updates will not occur).
+
+   A. You can use the URL of the official Percona's Version Service (default).
+      Set ``versionServiceEndpoint`` to ``https://check.percona.com/versions``.
+
+   B. Alternatively, you can run Version Service inside your cluster. This
+      can be done with the ``kubectl`` command as follows:
+      
+      .. code:: bash
+      
+         kubectl run version-service --image=perconalab/version-service --env="SERVE_HTTP=true" --port 11000 --expose
+
+   .. note:: Version Service is never checked if automatic updates are disabled.
+      If automatic updates are enabled, but Version Service URL can not be
+      reached, upgrades will not occur.
+
+#. Use the ``schedule`` option to specify the update checks time in CRON format.
+
+The following example sets the midnight update checks with the official
+Percona's Version Service:
+
+.. code:: yaml
+
+   spec:
+     updateStrategy: SmartUpdate
+     upgradeOptions:
+       apply: 5.7.27-31.39
```

Comment:
I would here maybe better use "Latest" or "Recommended" since it makes more sense for repeated checking for new versions.